Backend/src/v1/testing.py

```diff
@@ -0,0 +1,131 @@
+# own
+from datetime import time
+import os
+import random
+from ..types import TestingResponse, DatasetsResponse
+from ..utils import (
+    download_qaqc_source_dataset as util_download_qaqc,
+    get_qaqc_info_dict as util_get_info_dict,
+)
+from .dataset import upload_dataset, get_all_processed_datasets
+from .data import get_all_labels, get_all_nodes
+from .run_ml import run, get_status
+
+#pip
+from fastapi import APIRouter, HTTPException, UploadFile
+from fastapi.testclient import TestClient
+
+import asyncio
+import time
+import io
+import json
+
+
+
+router = APIRouter(
+    prefix="/testing",
+    tags=["Testing"],
+)
+
+@router.post("/")
+async def manual_test_sequence(
+    filename: str,
+    dim_red_method: str,
+    n_rows: int | None = None,
+    run_time: float | None = None,
+    run_unit: str = "s", 
+    ) -> TestingResponse:
+    """
+    Run a test sequence of following endpoints: 
+
+    0. Downloads the source QAQC data set.
+    1. Uploads n_rows of the source data.
+    2. Starts an async run of the ml lib.
+    3. GET running statuses.
+    5. GET Processed data sets.
+    6. Get labels.
+    7. Get 10 random nodes of from the generated collection.
+
+    Returns a response consisting of a collection of status responses from other routes ran in the sequence. 
+
+    Parameters:
+
+    1. filename, will name the generated collection.

```

Comment:
Something is wrong with the swagger formatting of this part. The numbers are not included

Reply:
Formatting is on point for me.. cant see any problems

![image](https://github.com/user-attachments/assets/0c0488df-ff5e-49ef-9ebb-cc6e57c91816)